packages/conf-assimp/conf-assimp.1/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "geoffderosenroll@gmail.com"
+authors: ["Geoff deRosenroll"]
+homepage: "https://github.com/assimp/assimp"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "BSD-3-Clause"
+build: [
+  ["assimp" "--help"]
+]
+depexts: [
+  ["assimp-utils" "libassimp-dev"] {os-family = "debian"}
+  ["assimp-utils" "libassimp-dev"] {os-family = "ubuntu"}
+  ["assimp"] {os-family = "arch"}
+  ["assimp-devel"] {os-distribution = "fedora"}
+  ["assimp-devel"] {os-distribution = "ol"}
+  ["assimp" "libassimp-dev"] {os-family = "suse"}
+  ["lib64assimp-devel"] {os-distribution = "mageia"}
+  ["assimp"] {os-family = "alpine"}
+  ["assimp"] {os-distribution = "nixos"}
+  ["assimp"] {os = "openbsd"}
+  ["assimp"] {os = "macos" & os-distribution = "homebrew"}
+  ["assimp"] {os = "macos" & os-distribution = "macports"}
+  ["assimp"] {os = "freebsd"}
+  ["assimp"] {os = "win32" & os-distribution = "vcpkg"}
```

Comment:
Side note: opam doesn't support vcpkg currently so this part isn't used.

Reply:
I suspected that might be the case, but thought the message might be useful to the user anyway. Thanks for confirming.